api/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,70 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"reflect"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+const (
+	// Group is the group of the objects
+	Group = "cloudscale.crossplane.io"
+
+	// Version is the version of the objects
+	Version = "v1alpha1"
+)
+
+var (
+
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: Group, Version: Version}
```

Comment:
Note: we have traditionally separated out like resources into their own groups (i.e. an S3 bucket would live in the `storage.cloudscale.crossplane.io` group). I do not think this pattern needs to necessarily be followed verbatim, but it may be helpful as you begin to support more resources.